modules/appeals_api/spec/fixtures/valid_200995.json

```diff
@@ -0,0 +1,39 @@
+{
+  "data": {
+    "type": "supplementalClaim",
+    "attributes": {
+      "benefitType": "compensation",
+      "veteran": {
+        "phone": {
+          "areaCode": "555",
+          "phoneNumber": "8001111"
+        },
+        "email": "josie@example.com",
+        "timezone": "America/Chicago"
+      },
+      "evidenceSubmission": {
```

Comment:
What if there is no evidence submission? What if the Veteran simply lists the locations and dates of evidence? Are we planning on making evidence submission a requirement? Sorry if I missed this!

Reply:
Not sure on this either... Just implementing the draft schema that Jack created.

Reply:
I'll bring these questions up at standup today 🎉

Reply:
If the evidenceSubmission type is `uploaded`, we're expecting an upload. If it's `retrieved`, then we expect `retrieveFrom`.

Reply:
I think the word `retrieved` is throwing me off. The idea is that if it's not uploaded, then it is _to be_ retrieved, but hasn't yet been retrieved. Can we call this `retrieve` or `to_retrieve` or `retrieve_from`?

Reply:
the word `retrieved` is in reference to the "type of evidence submitted with the supplemental claim". Is it 'retrieved' evidence, or is it 'uploaded' evidence? I don't think that specific param is in reference to anything else. I think we can work with the `retrieveFrom` array name a bit more, though...

Reply:
I think this is where I'm confused: `LIST BELOW ANY VA MEDICAL CENTER(S) (VAMC), VA TREATMENT FACILITIES, OR FEDERAL DEPARTMENTS OR AGENCIES THAT HAVE NEW AND RELEVANT
EVIDENCE THAT YOU ARE AUTHORIZING VA TO OBTAIN IN SUPPORT OF YOUR SUPPLEMENTAL CLAIM`

Reply:
The claimant fills out the name and location section, asking that the VA obtain the info...thus, to be retrieved. Are we not doing it this way?

Reply:
I vote `upload` and `retrieve`.

Reply:
I think it's important to separate the "supplemental claim 'process'" from "supplemental claim 'data'". The schema in this case is talking about 'what type of data to expect in order to create a supplemental claim. The schema is asking - is the evidence 'uploaded', 'retrieved', or both? 

Once we know that, we then ask for the `retrieveFrom` data, which can be `dataToBeRetrieved` or some similar configuration, informing the process (and where the form deals with the future process').